src/AzerothWarsCSharp.MacroTools/Spells/ElectricStrike.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using AzerothWarsCSharp.MacroTools.SpellSystem;
+using static War3Api.Common;
+
+namespace AzerothWarsCSharp.MacroTools.Spells
+{
+  public sealed class ElectricStrike : Spell
+  {
+    public int STUN_ID { get; init; } = FourCC("A0RD"); //The ability that gets cast on each unit in the radius
+    public int PURGE_ID { get; init; } = FourCC("Aprg"); //The ability that gets cast on each unit in the radius
+    public string PURGE_ORDER { get; init; } = "purge";
+    public string STUN_ORDER { get; init; } = "firebolt";
+    private float RADIUS { get; init; } = 200.00F;
```

Comment:
Make RADIUS and EFFECT public parameters and configure them correctly in SpellSetup.cs.